gas/testsuite/gas/riscv/insn-dsp64.d

```diff
@@ -0,0 +1,91 @@
+#as: -mext-dsp -march=rv64gc
+#objdump: -dr -Mstandard
+
+.*:[ 	]+file format .*
+
+
+Disassembly of section .text:
+
+0+000 <dsp64>:
+[ 	]+.*:[ 	]+.*[ 	]+add32[ 	]+r1,r2,r3
+.*R_RISCV_RELAX_ENTRY.*
```

Comment:
We don't have this relocation.  If you need new relocations, you should file an issue to riscv-psabi first, we need the approvals of riscv community, and developers of gnu and llvm.

Reply:
Test cases are updated in the latest commit. Thanks for spotting it out!